### DISCUSSION TITLE
Can I disable downloading remote media?

### DISCUSSION THREAD

**Discussion by lzc256** [mastodon/mastodon#26270] [Q&A] [answered]
I am running a mastodon instance on my server, but it keeps downloading remote media, over 5 gb per day. 

I know that Mastodon has to download remote media for faster browsing experience, but I have to pay $0.1 for every 1GB data transferred, and the disk is also running out. The price is just too high.

So I am looking for a solution. Can I configure the instance to fetch toots only, and never download remote media until it is requested (eg. When a toot is viewed, fetch its media from remote server)?

Expecting reply.

**Comment by p37307** [comment]
This has been suggested and discussed at 

https://github.com/mastodon/mastodon/issues/15195 and  https://github.com/mastodon/mastodon/issues/15193

**Comment by renchap** [comment, marked as answer]
This is not supported at the moment and would require major changes in many places to support it.
